### DISCUSSION TITLE
DNS Verification Issue

### DISCUSSION THREAD

**Discussion by mrkingsleyobi** [postalserver/postal#1713] [Help with using Postal] [answered]
I have install postal correctly and I can log in. Adding and setting up my domain
![screencapture-ap-www-namecheap-Domains-DomainControlPanel-lightmailer-com-advancedns-2021-11-16-12_23_14](https://user-images.githubusercontent.com/4938148/141979557-3cb11d6a-2e21-46f4-b54f-6383e51150fc.png)

I got this 

![screencapture-postal-lightmailer-org-lightmailer-servers-lightmailer-server-domains-0afd2341-dcf1-49b5-96a7-ae29332361b3-setup-2021-11-16-12_37_24](https://user-images.githubusercontent.com/4938148/141979599-a167e0c2-2eb7-4ac2-8716-6b94420bcdcd.png)

I also comfirmed with mxtoolbox dnslookup I got this

![Screen Shot 2021-11-16 at 12 40 33 PM](https://user-images.githubusercontent.com/4938148/141979681-e12e17d0-a07e-4b27-aa33-4414257a81f3.png)

I need help.

**Comment by willpower232** [comment, marked as answer]
I think the Supertool was upset that you gave it lightmailer.com and not postal.lightmailer.com but it looks like your DNS is correct. Is Postal happy with it now? There might have just been a delay in your server being able to see the updated records.
